src/qml/FeatureForm.qml

```diff
@@ -690,7 +690,7 @@ Page {
 
             anchors { right: parent.right; top: constraintDescriptionLabel.bottom; verticalCenter: menuButton.verticalCenter }
 
-            onCheckedChanged: {
+            onClicked: {
```

Comment:
Why the change here? Just want to learn why so? Because the "changed" one will trigger from the initial qgis values?

Reply:
@suricactus , yeah, if we used oncheckedchanged, QField (rightfully) complaints about a binding loop.

Reply:
(and that's because clicking on the toolbar button triggers the oncheckedchanged signal, and in there we modify the RememberValue property which in turn re-triggers an oncheckedchanged signal as the checked property is re-evaluated).